.github/workflows/cmake.yml

```diff
@@ -0,0 +1,29 @@
+# Note: CMake support is community-based. The maintainers do not use CMake
+# internally.
+
+name: cmake
+
+on:
+    push:
+    pull_request:
+        branches:
+            - main
+
+permissions:
+    contents: read
+
+jobs:
+  cmake_min:
+    runs-on: ubuntu-latest
```

Comment:
Can we include the platform in the job's name? Just "linux" is probably sufficient.